packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -727,8 +736,9 @@ class AnimationController extends Animation<double>
     }());
     assert(max >= min);
     assert(max <= upperBound && min >= lowerBound);
+    assert(repeatTimes == null || repeatTimes > 0, 'Repeat times shall be greater than zero if not null');
     stop();
-    return _startSimulation(_RepeatingSimulation(_value, min, max, reverse, period!, _directionSetter));
+    return _startSimulation(_RepeatingSimulation(_value, min, max, reverse, period!, _directionSetter, repeatTimes,));
```

Comment:
```suggestion
    return _startSimulation(_RepeatingSimulation(_value, min, max, reverse, period!, _directionSetter, repeatTimes));
```

(nit: only use a trailing comma when arguments are on separate lines)